samples/java-eventsourced-shopping-cart/src/main/java/com/example/shoppingcart/Main.java

```diff
@@ -20,7 +20,8 @@ public static AkkaServerless createAkkaServerless() {
     // If you prefer, you may remove this and manually register these components in a
     // `new AkkaServerless()` instance.
     return AkkaServerlessFactory.withComponents(
-        ShoppingCart::new);
+        ctx -> new ShoppingCart()
```

Comment:
We have tried to standardize on that the constructor takes a context parameter even though it's not used. That is what we generate and good to show that possibility in samples (and users can remove if they don't like it)

Reply:
I see that becomes a bit of a problem for the tests:
```
ShoppingCartTestKit testKit = new ShoppingCartTestKit(new ShoppingCart());
```

Can we create a test context for them, like:
```
ShoppingCartTestKit testKit = ShoppingCartTestKit.of(ctx -> new ShoppingCart());
```

which then can be written with `ShoppingCart::new` there also